index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=PT+Serif&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=PT+Serif&family=Poppins:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/main.css">
+  <title>My team page</title>
+</head>
+<body>
+  <main class="container">
+    <h1 class="creative-crew-title">The creative crew </h1>
+    <h3 class="we-are-title">Who we are</h3>
+    <p class="information-team">We are team of creatively diverse. driven. innovative individuals working in various locations from the world.</p>
```

Comment:
If follow the pattern this class needs to be called `team-information`  or `team-information-text`  ?